CHANGES.txt

```diff
@@ -1,9 +1,12 @@
 5.1
+ * Increasing default for auto_repair.sstable_upper_threshold considering large Cassandra tables & revert three lines removed from CHANGES.txt due to a merge mistake (CASSANDRA-20586)
```

Comment:
Don't need to have the note about the changelog issues (I will remove on commit)